### PR TITLE
edit model created like component created

### PR DESCRIPTION
**Pull request by Rockyraven**
feat:
created Edit Model
like Functionality

### CODE REVIEW

File: src/pages/profile-page/Profile.js [hunk 1]
```diff
@@ -1,113 +1,133 @@
...
-              Logout
-            </button>
-          </div>
-        )}
+          )}
+       
+        <hr />
+        <button
+        style={{isActive: "blue", color: "red"}}
+        //  style={({ isActive }) => (isActive ? enableStyle : disabledStyle)}
```
Comment (Shubhamkr2610) [inline src/pages/profile-page/Profile.js:116]: remove unused code

File: src/component/modal/EditModal.jsx [hunk 2]
```diff
@@ -25,17 +25,30 @@ export const EditModal = ({ _id }) => {
...
   };
+  const editHandler = () => {
+    dispatch(editPost({ token: encodedToken, postId: _id, text: text}));
+    handleClose();
+  };
+
+  
+  const openModel = () => {
+    // dispatch(fetchPost());
+    handleOpen();
```
Comment (Shubhamkr2610) [inline src/component/modal/EditModal.jsx:46]: remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]
gave some suggestion  ,  do refactor them . then you can proceed for merge

File: src/component/modal/EditModal.jsx [hunk 3]
```diff
@@ -21,48 +20,44 @@ const style = {
   p: 4,
 };
 
-
-export const EditModal = ({_id}) => {
+export const EditModal = ({ _id }) => {
   const [open, setOpen] = useState(false);
   const handleOpen = () => setOpen(true);
```
Comment (arjun-computer-geek) [inline src/component/modal/EditModal.jsx:25]: Suggestion : you can do open and close with one state.

**Review by arjun-computer-geek — APPROVED** [review]
Nothing so much. A suggestion is given. Please consider if you find this important. Else codes are good and readable .

File: src/component/modal/EditModal.jsx [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
 import Box from "@mui/material/Box";
 import Modal from "@mui/material/Modal";
-import Button from "@mui/material/Button";
 import Typography from "@mui/material/Typography";
 import EditOutlinedIcon from "@mui/icons-material/EditOutlined";
 import TextareaAutosize from "@mui/material/TextareaAutosize";
 import { height } from "@mui/system";
 import { useDispatch, useSelector } from "react-redux";
-import { fetchPost } from '../../redux/slices/postSlice';
+import { fetchPost } from "../../redux/slices/postSlice";
```
Comment (khushijohri2001) [inline src/component/modal/EditModal.jsx]: I would suggest you to use absolute paths and noth '../../'. It was giving an error while i was deploying it to Netlify then I had to change the imports

**Review by khushijohri2001 — COMMENTED** [review]
Otherwise everything looks good. Clean code!!

**Pull request merged**